.github/workflows/gitleaks.yml

```diff
@@ -0,0 +1,21 @@
+name: gitleaks
+on:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  scan:
+    name: gitleaks
+    runs-on: ubuntu-24.04
```

Comment:
shouldnt it be a new runner label (for example centreon-ubuntu-22.04) ?

Reply:
As it is a public repository, why should we use a private runner ?